WebAPI/LearningHub.Nhs.Database/Stored Procedures/Hierarchy/CheckCatalogueHasExternalReference.sql

```diff
@@ -0,0 +1,68 @@
+﻿-------------------------------------------------------------------------------
+-- Author       Sarathlal
+-- Created      29 August 2024
+-- Purpose      Return if Node has an external reference
+--
+-- Modification History
+--
+-- 29-08-2024  SS	Initial Revision.
+
+-------------------------------------------------------------------------------
+CREATE PROCEDURE [hierarchy].[CheckCatalogueHasExternalReference]
```

Comment:
It looks like this stored procedure relies on a node reference to an external catalogue.
If a single resource is referenced without any folders then this may not be picked up by the current logic

Reply:
@DRJTechnology This scenario has been handled. Please have a look.